dataset-snippets/measurement_by_age_and_gender.plotnine

```diff
@@ -0,0 +1,34 @@
+# Use snippet 'measurement_by_age_and_gender' to plot joined demographics and measurements dataframes.
+# This plot assumes 'measurement_df' was created using snippet 'Basic operations -> join_dataframes' to
+# join together demographics and measurements dataframes.
+# See also https://plotnine.readthedocs.io/en/stable/
+
+
+# There could be many different measurements in the dataframe. By default, plot the first one.
+measurement_to_plot = measurement_df.STANDARD_CONCEPT_NAME.unique()[0]
+
+# Create a derived variable for age group.
+measurement_df['age'] = (pd.to_datetime('today').year
+                                     - measurement_df.DATE_OF_BIRTH.map(lambda x: x.year))
+measurement_df['age_group'] = pd.cut(measurement_df['age'],
+                                                 [-np.inf, 34.5, 49.5, 64.5, np.inf],
+                                                 labels=["<35", "35-49", "50-64", "65+"])
```

Comment:
Any reason to diverge between Python and R on the age buckets? Don't feel strongly, mostly just curious.

Reply:
+1 for switching to bigrquery!!

I wrote the R version first. When I re-implemented it in pandas, it just looked really long, so I went with fewer groups to reduce the code.

For both of those, people will likely edit them to age groupings relevant to their particular analyses.